docs/Guide/getting-started/updating-from-v3-to-v4.mdx

```diff
@@ -0,0 +1,44 @@
+---
+sidebar_position: 5
+title: Updating from v3 to v4
+toc_max_heading_level: 5
+---
+
+## Before you start
+
+It is important to note that the primary breaking change of Sapphire v4 is support for DiscordJS v14.x. This means that
+for the bulk of the breaking changes we want to refer you to [Updating from v13 to v14][djs-migration].
+
+## Sapphire specific changes
+
+Sapphire has a few breaking changes that are not related to DiscordJS. These are:
+
+- The [`Identifiers`][Identifiers] enum was changed from `const enum` to [`enum`][enum].
+- The [`CommandOptionsRunTypeEnum`][CommandOptionsRunTypeEnum] enum was changed from `const enum` to [`enum`][enum].
+- The [`CommandPreConditions`][CommandPreConditions] enum was changed from `const enum` to [`enum`][enum].
+- The [`InteractionHandlerTypes`][InteractionHandlerTypes] enum was changed from `const enum` to [`enum`][enum].
+- The [`InternalRegistryAPIType`][InternalRegistryAPIType] enum was changed from `const enum` to [`enum`][enum].
```

Comment:
I wouldn't log this as its not exposed to users
And also iirc should still be const